helm/prometheus-rules/templates/platform/atlas/alerting-rules/prometheus-agent.rules.yml

```diff
@@ -51,6 +51,7 @@ spec:
         cancel_if_cluster_is_not_running_prometheus_agent: "true"
         cancel_if_cluster_status_creating: "true"
         cancel_if_cluster_status_deleting: "true"
+        cancel_if_cluster_has_no_workers: "true"
```

Comment:
This inhibition exists in the alertmanager config?

Reply:
Yes it does. Do you have reasons to doubt it?

Reply:
Not at all :) I know we have a lot of "inhibitions" configured that are not used there so I was wondering :)

Reply:
If you're looking for evidence, it's here:
https://github.com/giantswarm/prometheus-meta-operator/blob/main/files/templates/alertmanager/alertmanager.yaml#L612